conf/CFBG.conf.dist

```diff
@@ -27,8 +28,22 @@
 #       Description: Maximum players in party for enter queue
 #       Default: 3
 #
+#   CFBG.EvenTeams
+#        Description: Set Battleground invitation type.
+#        Default:     0 - (Normal, Invite as much players to battlegrounds as queued,
+#                          Don't bother with balance)
+#                     1 - (Don't allow to invite much more players of one faction.
+#                          Keep an even quantity of players in Battleground)
+#
+#   CFBG.EvenTeams.MaxPlayersThreshold
+#       Description: team member quantity until the EvenTeams rule is valid
+#       Default: 5 - (Quantity of players after the EvenTeams rule will be disabled in the BG)
```

Comment:
is this 5 per team or in total? It's not clear.

`Default: 5 - (Quantity of players per team after the EvenTeams rule will be ignored)`

Reply:
per team